src/application/index.js

```diff
@@ -52,7 +55,7 @@ const main = async () => {
       }
     }
   } catch (error) {
-    throwGithubError(error.message)
+    throwErrorFailOnHardFailure(error)
```

Comment:
instead of `throwErrorFailOnHardFailure` what about `outputFailure` name: 

```
  /** If app fails outputs a Warning or either throws an Error depending on hard-failure env **/
outputFailure(error)
```